.github/workflows/kotlin.yml

```diff
@@ -151,3 +151,9 @@ jobs:
           api-level: ${{ matrix.api-level }}
           arch: x86_64
           script: ./gradlew connectedCheck --build-cache --no-daemon --stacktrace
+
+      - name: Upload results
+        uses: actions/upload-artifact@v2
+        with:
+          name: insrumentation-test-results
+          path: ./**/build/reports/androidTests/connected/**
```

Comment:
Where can we see them?

Reply:
![Screenshot_20200512-091610](https://user-images.githubusercontent.com/101754/81719128-80986200-9431-11ea-9bca-35e87db2e88b.png)